src/nft/components/details/detailsV2/DataPageTraits.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components/macro'
+
+import { containerStyles } from './shared'
+
+const TraitsContainer = styled.div`
+  height: 528px;
+  padding: 16px 20px;
+
+  ${containerStyles}
+`
+
+export const DataPageTraits = () => {
+  return <TraitsContainer>Traits</TraitsContainer>
```

Comment:
I think it might actually be better to put this in a Tabbed component so it can share the same styling as the others. Just make it an array of 1